microsoft/testsuites/display/drm_sanity.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import Any, cast
+
+from assertpy import assert_that
+
+from lisa import (
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+
+from lisa.base_tools import Cat
+from lisa.tools import Lsmod
+
+@TestSuiteMetadata(
+    area="drm",
+    category="functional",
+    description="""
+    This test suite uses to verify drm driver sanity.
+    """,
+    requirement=simple_requirement(unsupported_os=[]),
+)
+class DRM_Sanity(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This case is to check whether the drm driver registered successfully.
+
+        Step,
+        1. Check hyperv_drm driver is inserted correctly.
+        """,
+        priority=0,
+        use_new_environment=True,
+    )
+    def verify_drm_driver(self, node: Node, log: Logger) -> None:
+        lsmod = node.tools[Lsmod]
+        if lsmod.module_exists("hyperv_drm"):
+            log.info(f"hyperv_drm module present")
+            ret  = "true"
+        else:
+            log.error(f"hyperv_drm module absent")
+            ret = "false"
+        assert_that(ret).is_equal_to("true")
```

Comment:
Give the error message, once it's failed, it should let people know what's expected.